slugify.js

```diff
@@ -0,0 +1,93 @@
+require('isomorphic-fetch');
+
+const tsProjectId = process.env.TS_PROJECT_ID;
+const tsApiKey = process.env.TS_API_KEY;
+const shape = process.env.TS_SHAPE;
+const origin = 'title';
+const target = 'slug';
+
+const endpoint = `https://api.takeshape.io/project/${tsProjectId}/graphql`
+
+const slugify = async () => {
+  console.log('loading...');
+  const data = await getShapeList();
+  const items = data[`get${shape}List`].items;
```

Comment:
Put this line inside `getShapeList` so that only that function needs to know about how to format the query name.

Reply:
Also added a validation for when there's no returned data.